src/components/ec-option-card/ec-option-card.story.ts

```diff
@@ -0,0 +1,114 @@
+import { IconName } from '../ec-icon/iconNames';
+import EcOptionCard from './ec-option-card.vue';
+import type { OptionCardProps } from './types';
+
+export default {
+  title: 'Option Card',
+  component: EcOptionCard,
+};
```

Comment:
could you please add here argTypes similar as done in ec-input-field for the default story. This way we'll have correct controls in the storybook for option-card component.